_posts/minutes/2016-04-22-22.md

```diff
@@ -0,0 +1,10 @@
+---
+    title: 'Minutes of Meeting on 22/04/2016'
+    author: qaisjp
+    category: minutes
+---
+This week's meeting covers details relating to sponsors.
+
+**Present**:
+
+Lisa Xie, Vesko Stefanov, Viktorija Lukosiute, Benedetta Mussati, Susanne Koster, Martynas Grigonis
```

Comment:
Oh, that's a good question. It can cut both ways, that one, on the one hand, they could be unhappy about what they're not getting, but it might also encourage them to give more. I think the Imperial equivalent of CompSoc gets like £7k from M$, it's ridiculous. Either you remove it, or you add a note saying 'nothing of importance was discussed', Idk.